packages/panzer/dof-mgr/src/Panzer_DOFManager_impl.hpp

```diff
@@ -930,7 +938,10 @@ DOFManager<LO,GO>::buildTaggedMultiVector(const ElementBlockAccess & ownedAccess
 
     // temporary working vector to fill each row in tagged array
     std::vector<int> working(overlap_mv->getNumVectors());
-    ArrayRCP<ArrayRCP<GO> > edittwoview = overlap_mv->get2dViewNonConst();
+    auto dual_view = overlap_mv->getDualView();
```

Comment:
I would prefer that you not call `getDualView`.  Please call the functions that get the Kokkos::View of the appropriate memory space directly.  This is fine for now, but I do plan on getting rid of `getDualView` at some point, since it exposes an implementation detail that constrains Tpetra to have allocations on both sides at all times (it can't do lazy allocation).

Reply:
ok. Will fix that as well.